spammers.txt

```diff
@@ -63,6 +63,7 @@
 @hanoverit.co.uk OR 
 @haverynash.com OR 
 @hays.com OR 
+@humancapitalconsulting.ro OR
```

Comment:
@wyewye you need to add a *space* to the end of any line addition you are proposing. 
(*as per instructions in the contributing guide* ... https://github.com/drcongo/spammy-recruiters/blob/master/CONTRIBUTING.md format: `@domain[space]OR[space]`

Also its a good idea to include a *sample* [redacted](https://www.google.co.uk/#q=redacted) email you received from the  "*spammer*" so there is a record of their undesirable email ...